test/ChainRulesTests.jl

```diff
@@ -0,0 +1,137 @@
+module ChainRulesTest
+
+using LinearAlgebra
+using ChainRules
```

Comment:
This hides the problem described above. Maybe it would be good to not load ChainRules but only ChainRulesCore?